packages/pricing/src/joiner-config.ts

```diff
@@ -32,6 +32,18 @@ export const joinerConfig: ModuleJoinerConfig = {
     {
       name: "price_sets",
     },
+    {
+      name: "calculated_prices",
+      args: {
+        methodOverride: "calculatePrices",
+      },
+    },
+    {
+      name: "calculated_price",
+      args: {
+        methodOverride: "calculatePrice",
+      },
+    },
```

Comment:
thought: Are we sure this is how we want to do it?

I think for the first iteration and to not rush into a pattern, the call to calculate prices should be done separately from the Remote Query call. 

So right now, we will only use the Remote Query engine as a way to collect data for further processing.